icu4c/source/common/rbbitblb.cpp

```diff
@@ -28,6 +28,8 @@
 
 U_NAMESPACE_BEGIN
 
+const int32_t kStateThresholdFor8BitsTable = 127;
```

Comment:
why is this 127 and not 255? there is no dict bit in the next-state value, is there?

Reply:
ping -- still 127 but I don't understand why